### ISSUE TITLE
Add term thickened cortex of the fibula

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10486]
Preferred term label:
Thickened cortex of the fibula

Synonyms
Broad cortex of fibula - Cortical thickening of the fibula - Thickened cortices of fibula

Definition (free text, please give PubMed ID)
Abnormal thickening of the cortex of the fibula
PMID24389367

Parent term (use hpo.jax.org/app)
Thickened cortex of long bones HP:0000935
Abnormal fibula morphology HP:0002991

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Caffey Disease (omim114000), for example: PMID24390061
Camurati-Engelmann disease (omim131300), for example: PMID15326622

Your nano-attribution (ORCID)
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw @azankl Adding new term: 
- ID: HP:6000705
- label: Thickened fibular cortex
- parents: [Abnormal fibula morphology (HP:0002991)](https://hpo.jax.org/app/browse/term/HP:0002991) | [Thickened cortex of long bones (HP:0000935)](https://hpo.jax.org/app/browse/term/HP:0000935)
- pmid: PMID:24389367
- def: Abnormal increase in the thickness (distance between the inner and outer boundaries) of the outer layer (cortex) of the fibula.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10486
- synonyms: Cortical thickening of the fibula(NONE)|Thickened cortices of fibula(NONE)|Broad cortex of fibula(NONE)|Thickened cortex of the fibula(NONE)
